src/oge/helpers.py

```diff
@@ -174,17 +202,11 @@ def create_plant_ba_table(year):
         plant_states, how="left", on="plant_id_eia", validate="m:1"
     )
 
-    # convert the dtype of the balancing authority code column from string to object
-    # this will allow for missing values to be filled
-    plant_ba["balancing_authority_code_eia"] = plant_ba[
-        "balancing_authority_code_eia"
-    ].astype(object)
-    plant_ba["balancing_authority_code_eia"] = plant_ba[
-        "balancing_authority_code_eia"
-    ].fillna(value=np.NaN)
-
     # load the ba name reference
-    ba_name_to_ba_code = pd.read_csv(reference_table_folder("ba_reference.csv"))
+    ba_name_to_ba_code = pd.read_csv(
+        reference_table_folder("ba_reference.csv"),
+        dtype={"ba_name": "string", "ba_code": "string"},
```

Comment:
should we just use `column_checks.get_dtypes()` here?

Reply:
`ba_name` is not in `get_types()`. Should I add it?

Reply:
No, this is fine. I generally like using our built in function since we have it, but I don't think that there's really a benefit to using it here.